### DISCUSSION TITLE
usecase 라벨 제거 건의

### DISCUSSION THREAD

**Discussion by scarf005** [jiphyeonjeon-42/frontend#468] [Ideas]
![image](https://github.com/jiphyeonjeon-42/frontend/assets/54838975/b789f504-0ab1-47d2-b05c-19f2d2a5cb6e)

1. usecase-1 대신 '검색' 등 조금 더 명확한 이름의 제목을 사용하면 라벨을 구분하는데 도움이 될 것 같습니다.
2. 현재 usecase-* 라벨을 사용하는 이슈가 없어 제거하더라도 큰 문제가 없을 것 같습니다.

**Comment by not-using** [comment]
동의합니다! 라벨이 완전 초기 1차 MVP 완성하기 전에 만들어놓고 관리를 못했었네요
어떤 식으로 라벨을 다시 정의하면 좋을까요?

**Comment by chanhihi** [comment]
저는 엄청 동의 합니다.
